products/PDFmorph.rst

```diff
@@ -0,0 +1,120 @@
+.. include:: ../abbreviations.txt
+
+########
+PDFmorph
+########
+
+Overview
+========
+
+.. note::
+	PDFmorph is an application which makes use of the Command Line Interface. If
+	you are unfamiliar with the CLI, it is recommended that you consult online
+	resources and become somewhat familiar before using PDFmorph.
+
+PDFmorph is a Python software package designed to increase the insight researchers
+can obtain from measured atomic pair distribution functions (PDFs) in a model 
+independent way. The program was designed to help a researcher answer the question: 
+"has my material undergone a phase transition between these two measurements?"
+
+PDFmorph makes use of several data manipulation techniques to correct for benign
+effects such as thermal expansion and increased thermal motion before computing and
+plotting a difference curve between two PDFs. One PDF is identified as the "target" 
+PDF and the second is "morphed" by "stretching", "smearing", and "scaling". PDFmorph
+will vary amplitude of morphing transformations to obtain the best fit between
+morphed and target PDFs, then plot them along with the difference curve plotted
+below.
+
+There are also a few other morphing transformations in the program.
+
+Finally, we note that PDFmorph should work on other spectra that are not PDFs, 
+though it has not been extensively texted beyond the PDF.
+
+PDFmorph will soon be included in :doc:`xPDFsuite <xPDFsuite>`, a powerful
```

Comment:
remove this paragraph for now.  Not sure if there is a nice way to comment it out so we can add it back later.

Reply:
will do!